app/src/main/res/layout/item_lot.xml

```diff
@@ -53,10 +43,7 @@ android:layout_marginTop="20dp"
         android:id="@+id/text_id_lot"
         android:layout_width="wrap_content"
         android:layout_height="31dp"
-
-
         android:textAppearance="@style/some_id"
-
         android:textColor="#546E7A"
```

Comment:
Fetch the value from resource/values/colors

Reply:
ok, do you mind if I give a moderate priority to this part?